From 258cda0080f60197f09bb924a5c73bf833e099cc Mon Sep 17 00:00:00 2001
From: Sinthu Raja <sinthu.raja@ti.com>
Date: Tue, 27 Jul 2021 18:08:35 +0530
Subject: [PATCH 7/9] arm64: dts: ti: k3-j721e: Update compatible for DPHY

Use the j721e specific compatible for DPHY. This allows to use the
j721e specific driver quirks

Signed-off-by: Sinthu Raja <sinthu.raja@ti.com>
---
 arch/arm64/boot/dts/ti/k3-j721e-main.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/ti/k3-j721e-main.dtsi b/arch/arm64/boot/dts/ti/k3-j721e-main.dtsi
index d8398eb14371..7e63482ca0ee 100644
--- a/arch/arm64/boot/dts/ti/k3-j721e-main.dtsi
+++ b/arch/arm64/boot/dts/ti/k3-j721e-main.dtsi
@@ -2430,14 +2430,14 @@ csi1_port4: port@4 {
 	};
 
 	dphy0: phy@4580000 {
-		compatible = "cdns,dphy";
+		compatible = "ti,j721e-dphy";
 		reg = <0x0 0x4580000 0x0 0x1100>;
 		#phy-cells = <0>;
 		power-domains = <&k3_pds 147 TI_SCI_PD_EXCLUSIVE>;
 	};
 
 	dphy1: phy@4590000 {
-		compatible = "cdns,dphy";
+		compatible = "ti,j721e-dphy";
 		reg = <0x0 0x4590000 0x0 0x1100>;
 		#phy-cells = <0>;
 		power-domains = <&k3_pds 148 TI_SCI_PD_EXCLUSIVE>;
-- 
2.31.1

